test/system/070-build.bats

```diff
@@ -29,6 +29,24 @@ EOF
     run_podman rmi -f build_test
 }
 
+@test "podman buildx - basic test" {
+    rand_filename=$(random_string 20)
+    rand_content=$(random_string 50)
+
+    tmpdir=$PODMAN_TMPDIR/build-test
+    mkdir -p $tmpdir
+    dockerfile=$tmpdir/Dockerfile
+    cat >$dockerfile <<EOF
+FROM $IMAGE
+RUN echo $rand_content > /$rand_filename
+EOF
+
```

Comment:
Shouldn't there be a `run_podman buildx ...` here?

Reply:
a little heavy on the cut and pasting.